### ISSUE TITLE
EVSS PCIU Migration: Test Plan Step 3 Form VA21p530 - FormProfile and FormProfile Spec

### ISSUE THREAD

**Issue by RachalCassity** [department-of-veterans-affairs/va.gov-team#81364]
### Test Plan: Step 3 Replace PCIU

 VAProfile::Configuration::SETTINGS.prefill  is set to true. PCIU data should no longer be used. 

`FormData.initialize_contact_information `prefills contact information with PCIU data.  
Coverband is showing a lot of data. This will have to be tested when migrating to VAProfile. PCIU references need to be removed. 

FormProfileSpec form data will need to be updated to use VAProfile data. PCIU data will be depreciated. 

```
  def initialize_contact_information
    opt = {}
    opt.merge!(vets360_contact_info_hash) if vet360_contact_info
    opt[:address] ||= user_address_hash
    opt[:email] ||= extract_pciu_data(:pciu_email)
    if opt[:home_phone].nil?
      opt[:home_phone] = pciu_primary_phone
      opt[:us_phone] = pciu_us_phone
    end
    format_for_schema_compatibility(opt)
    FormContactInformation.new(opt)
  end
```

## Tasks
- [x] Remove initialize_contact_information PCIU calls
- [x] In the settings.yml, add Settings.vet360.prefill = true
- [x] Update all form data to render VAProfile data

**Comment by RachalCassity** [comment]
Blocked by teams that own forms calling FormProfile

**Comment by jennb33** [comment]
10/10/2024 update - @RachalCassity is working on this

**Comment by jwoodman5** [comment]
@RachalCassity Finishing PR and doing some testing.

**Comment by jwoodman5** [comment]
@RachalCassity making a ticket for address validation. Will finish that then revisit this.

**Comment by jwoodman5** [comment]
Removing the tasks listed below and closing. @RachalCassity is creating a new issue for these outstanding tasks for next sprint.


- [ ] Remove VAProfile::Configuration::SETTINGS.prefill = true

- [ ] Remove VAProfile::Configuration::SETTINGS.prefill = false
